python/arcticdb/adapters/s3_library_adapter.py

```diff
@@ -244,3 +278,13 @@ def _configure_aws(self):
     @property
     def path_prefix(self):
         return self._query_params.path_prefix
+
+    
+    def copy_config_library_native_config_to_library(self, lib_cfg: LibraryDescriptor):
```

Comment:
I think this is a sign that something is a bit wrong with the design. We only support a single native configuration per Arctic object, so why are we storing them in the `_native_cfg` map as if we support many of them? I still don't really see why we need the `_native_cfg` map at all.